src/components/state-details.js

```diff
@@ -805,12 +753,14 @@ export default function StateDetailsPage ({ location, data }) {
       <hr className="mt-7" />
 
       <section className="text-center mb-8">
-        <div className="h1 mt-7 font-weight-bold">And that's it! 🎉</div>
+        <div className="h1 mt-7">✅</div>
+        <br className="d-none d-lg-block" />
+        <div className="h1 font-weight-bold">Ready to do your part?</div>
```

Comment:
Can you add some top spacing? This feels really close to the map.

![image](https://user-images.githubusercontent.com/3187531/177678960-9fcb63a2-6947-492b-a7be-632cd5fd13c6.png)

Reply:
yup